src/Raven.Server/Documents/Indexes/Static/JavaScriptIndex.cs

```diff
@@ -209,7 +210,8 @@ private HashSet<CollectionName> ExecuteCodeAndCollectReferencedCollections(strin
         private (List<string> Maps, List<HashSet<CollectionName>> MapReferencedCollections) InitializeEngine(IndexDefinition definition)
         {
             _engine.SetValue("load", new ClrFunctionInstance(_engine, "load", LoadDocument));
-
+            _engine.SetValue("getMetadata", new ClrFunctionInstance(_engine, "getMetadata", GetMetadata));
+            _engine.SetValue("id", new ClrFunctionInstance(_engine, "id", GetDocumentId));
```

Comment:
We need to go over the other functions we have for JS usage elsewhere and see what else we need here.

Reply:
There is a issue [RavenDB-13732](https://issues.hibernatingrhinos.com/issue/RavenDB-13894)
To add more functionality
I added id() and getMetadata() as got requested on google groups.